build.sbt

```diff
@@ -2,29 +2,122 @@ import com.typesafe.tools.mima.core._
 import explicitdeps.ExplicitDepsPlugin.autoImport.moduleFilterRemoveValue
 import org.http4s.sbt.Http4sPlugin._
 import org.http4s.sbt.ScaladocApiMapping
-
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.remote.server.DriverFactory
+import org.openqa.selenium.remote.server.DriverProvider
+import org.openqa.selenium.chrome.ChromeDriver
+import org.openqa.selenium.chrome.ChromeOptions
+import org.openqa.selenium.firefox.FirefoxOptions
+import org.openqa.selenium.firefox.FirefoxProfile
+import org.scalajs.jsenv.selenium.SeleniumJSEnv
+import sbtcrossproject.{CrossProject, CrossType, Platform}
+import java.util.concurrent.TimeUnit
 import scala.xml.transform.{RewriteRule, RuleTransformer}
 
+import JSEnv._
+
 // Global settings
 ThisBuild / crossScalaVersions := Seq(scala_213, scala_212, scala_3)
 ThisBuild / scalaVersion := (ThisBuild / crossScalaVersions).value.filter(_.startsWith("2.")).last
 ThisBuild / baseVersion := "1.0"
 ThisBuild / publishGithubUser := "rossabaker"
-ThisBuild / publishFullName   := "Ross A. Baker"
-
+ThisBuild / publishFullName := "Ross A. Baker"
+
+ThisBuild / githubWorkflowUseSbtThinClient := false
+ThisBuild / githubWorkflowBuildPreamble ++=
+  Seq(
+    WorkflowStep.Use(
+      UseRef.Public("actions", "setup-node", "v2.1.5"),
+      name = Some("Setup NodeJS v16"),
+      params = Map("node-version" -> "16"),
```

Comment:
Node 16 (experimentally) includes a `WebCrypto` API identical to the one in browsers. This let me quickly implement all the crypto methods in a cross-platform fashion. Node 16 will be the LTS in a few months I think.

Reply:
Also, it's worth mentioning: because JS imports these dynamically, people can still use Node 14 as long as they don't rely on any of these crypto APIs. TLS is still available either way, doesn't depend on Web Crypto.